app/src/main/java/edu/grinnell/appdev/grinnelldirectory/LoginActivity.java

```diff
@@ -2,12 +2,29 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
 
```

Comment:
You've done some very good documentation in your methods and it'll be nice to extend that here too! Let's document this class :)